src/screens/StoreEditScreen.js

```diff
@@ -9,7 +9,7 @@ class StoreEditScreen extends Component {
   }
 
   navigateTo = () => {
-    this.props.navigation.navigate('storeAdminList');
+    this.props.navigation.goBack();
```

Comment:
make sure that we are navigating to the correct view.

Reply:
yes, that is navigating to correct view